paper/paper.md

```diff
@@ -25,13 +26,18 @@ authors:
   - name: Seth D. Axen
     orcid: 0000-0003-3933-8247
     affiliation: 3
+  - name: Adrian Seyboldt
```

Comment:
Since your contribution was more extensive than mine, I'd suggest you be placed before me in the author list.

Reply:
Got to admit I'm pretty uncertain with all the conventions in publications. Thanks :-)